hasura/metadata/databases/default/tables/simpipe_runs.yaml

```diff
@@ -0,0 +1,36 @@
+table:
+  name: runs
+  schema: simpipe
+configuration:
+  custom_column_names: {}
+  custom_name: runs
+  custom_root_fields:
+    delete: delete_previous_run
+    insert: start_runs
+    insert_one: start_run
+    select: runs
+    select_by_pk: get_run
+object_relationships:
+- name: simulation
+  using:
+    foreign_key_constraint_on: simulation_id
+array_relationships:
+- name: steps
+  using:
+    foreign_key_constraint_on:
+      column: run_id
+      table:
+        name: steps
+        schema: simpipe
+select_permissions:
+- permission:
+    columns:
+    - simulation_id
```

Comment:
Isn't there overlapping columns between simpipe_runs and simpipe_steps? (ie, created, status, ended,started)

Reply:
It's the same columns and concept, but it shouldn't be the same data. The runs columns could probably be computed from the steps columns though, but that will make things a bit more complex. What do you think ?

Reply:
Yeah, on second thoughts it seems better to keep the timestamp columns separate as it is now. Maybe the run_started and run_ended would be a slightly longer interval than step1's and step_last's corresponding values.